core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -227,6 +228,8 @@ class SparkContext(config: SparkConf) extends Logging {
    */
   def getConf: SparkConf = conf.clone()
 
+  def getResources(): Map[String, ResourceInformation] = _resources
```

Comment:
we have a separate jira for this, I would remove this here because its not just getting the conf resources, there is also a discovery part of it for yarn/k8s.

Reply:
I‘m going to remove them.